App.tsx

```diff
@@ -77,7 +77,7 @@ export default function App() {
     return (
       <NavigationContainer>
         <Stack.Navigator
-          initialRouteName="Onboarding"
+          initialRouteName="Add Funds"
```

Comment:
please change to "Top up".
We are starting to play with wording based on feedback from claire and nirel. not mission critical ;-)
@Alphonce-Mutebi